integrations/flink_connector/README.md

```diff
@@ -59,13 +59,63 @@ mvn clean compile && mvn package
 cd ../sample-kinesis-to-timestream-app
 aws s3 cp target/sample-kinesis-to-timestream-app-0.1-SNAPSHOT.jar s3://YOUR_BUCKET_NAME/sample-kinesis-to-timestream-app-0.1-SNAPSHOT.jar
 ```
-3. Follow the steps in [Create and Run the Kinesis Data Analytics Application](https://docs.aws.amazon.com/kinesisanalytics/latest/java/get-started-exercise.html#get-started-exercise-7)
- - pick Apache Flink version 1.13.2
- - in "Edit the IAM Policy" step, add Timestream Write permissions to the created policy
+3. In the AWS Console, navigate to [Managed Apache Flink](https://console.aws.amazon.com/flink/home) to create the Apache Flink application
+ - Choose `Apache Flink Applications` in the navigation side bar
+ - Choose `Create streaming application`
+ - pick Apache Flink version 1.18
```

Comment:
Apache Flink version 1.19 is available via https://us-east-1.console.aws.amazon.com/flink/home?region=us-east-1#/create-flink-application,. can you pls check if the change can be updated to target 1.19. If this is too much work, then it can be taken in next iteration

Reply:
I tested upgrading to 1.19 and it will require additional work. I gave a quick look and with the deprecated interface [Sink.InitContext](https://javadoc.io/static/org.apache.flink/flink-core/1.19.0/org/apache/flink/api/connector/sink2/Sink.InitContext.html) it is not a quick fix. May be better to have this upgrade in another iteration, or I can expand the scope of this PR if you want the additional work done now.

Reference:
https://javadoc.io/static/org.apache.flink/flink-core/1.19.0/deprecated-list.html